src/natcap/invest/urban_nature_access.py

```diff
@@ -92,7 +92,7 @@
                     'type': 'number',
                     'units': u.meter,
                     'required':
-                        f'search_radius_mode == "{RADIUS_OPT_URBAN_NATURE}"',
+                        f'search_radius_mode.value == "{RADIUS_OPT_URBAN_NATURE}"',
```

Comment:
Similarly, this might not have the desired effect since I think CSV validation doesn't evaluate conditional requirement for columns: https://github.com/natcap/invest/blob/9e50106f64bd0525163e347030c4d439bb4b910f/src/natcap/invest/validation.py#L622-L623

Reply:
It works now!  As with vectors, at this point in the validity evaluation (at least when calling `validate`) the `required` attribute will be either True or False.

Since conditional requirement could be based on the value of other args, I didn't implement anything here in `get_validated_dataframe` because I believe we can now assume that conditional requirement has already been evaluated by a call to `validate`.  Maybe that's incorrect though ... what do you think?

Reply:
I think that's fine for now... `get_validated_dataframe` is used in some models outside of `validate`, but then this validation should've already happened.